Makefile

```diff
@@ -1,6 +1,15 @@
-# minimal Makefile to run Python unit tests
-# for flux-accounting
+FLUX_INSTALL_PREFIX ?= /usr
+FLUX=${FLUX_INSTALL_PREFIX}/bin/flux
+FLUX_PYTHON_VERSION=$(shell ${FLUX} python --version | cut -f 2 -d ' ' | cut -f 1-2 -d.)
+
+dependencies:
+	${FLUX} python -m pip install -r requirements.txt --user
+
+install: dependencies
```

Comment:
Sorry @cmoussa1, I know I suggested adding the `dependencies` target here so that they would be automatically installed on a `make install` call, but as I'm thinking more about system packaging, I'm not sure that is what we want.  If we package this up in an RPM, we won't want pip to install the dependencies in the user home directory.  To make packaging easier in the future, maybe we should drop this and require users to manually run `make dependencies` if they want the pip line to run.  Thoughts?

Reply:
No need to apologize @SteVwonder - your insight is really helpful! I think I'd agree with you. That change seems fine to me, so I'll go ahead and push a fix to remove `dependencies` from the `install:` target.